_posts/2023-03-17-introduction-to-discrete-event-simulations-in-healthcare/introduction-to-discrete-event-simulations-in-healthcare.Rmd

```diff
@@ -0,0 +1,112 @@
+
+At Providence, with the second Covid-19 pandemic dwindling, we wanted to test schedules to cut down the growing backlog of outpatient appointments required to fill while ensuring safe social distancing practices within the hospital.
+
+
+![Outpatient clinic appointment process at Providence Healthcare in August 2020](./providence_outpatient_process_map_aug2020.jpg)
+
+Under COVID regulations, all individuals are required to maintain social distancing between each other.  To follow the regulation and minimize unnecessary contacts, the Outpatient Service Department needs to control the maximum number of patients in each area of the department.  
+Planning to reopen more outpatient services (and to reduce the backlog created by shutdowns during the first two waves of the pandemic), the outpatient department wants to test adding additional appointments to their schedule with a combination of more staffing, longer hours, more patients and additional working days, while maintaining social distancing requirements.  
+
+
+The solution that was deployed for the PHC Outpatient Service Department consists of two parts:
+
+1) A simulation model that helps recreate the physical space capacity and analyze the system & process performance
+
+2) Designing a web user interface that requires users to input key features of the simulation for testing different scenarios. In addition, the model would output key performance indicators to help access the performance of the proposed plans/system. 
+
+The primary input that our user can manipulate to test out scenarios is the appointment schedule.  This appointment schedule include a breakdown of the types and quantities of appointments that would be seen in Inpatient services by the day of the week (Mon, Tue, Wed..).
+
+
+The simulation model consists of three main parts:
+
+- A defined object class called “Patient” with all necessary attributes and pre-defined calculation functions for results and outputs. Attributes include but not limit to: Patient index, appointment type, appointment clinic, probability of the patient needing wheel chair transportation from his/her home, and more.
+
+- The second part consists of logging, status update, capacity checking and calculation functions for the simulation process. Some of the process Detailed information of patient arrivals are calculated and recorded through these supporting functions.
+
+- The last part is the simulation model function, where it reads in user defined clinics’ capacity, registration time, and input data excel file. The model then reads the patient schedules and generates patients based on their scheduled appointment time. 
+
+
+
```

Comment:
How does the section below relate to the bullet points above?

Reply:
I think the content below could be cut as it gets real into the weeds. I think also drop the limitations as a section and note a limitation or two in a sentence, then provide some sort of conclusion (end on a high note)